sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -272,6 +290,46 @@ public EventData(ReadOnlyMemory<byte> eventBody) : this(eventBody, lastPartition
         /// <param name="pendingOwnerLevel">The producer owner level assigned to the event as part of a publishing operation.</param>
         ///
         internal EventData(ReadOnlyMemory<byte> eventBody,
+            IDictionary<string, object> properties = null,
```

Comment:
Please align the parameters for consistency with surrounding code:

```csharp
internal EventData(ReadOnlyMemory<byte> eventBody,
                   IDictionary<string, object> properties = null,
                   ...
```

Reply:
Fixed.